client/analysisui/iconsupport.css

```diff
@@ -0,0 +1,40 @@
+/*textbox sizing*/
+.jmv-option-icon[data-icon=ContinuousVariable] {
```

Comment:
at the moment we use lowercase enums in the yaml:

https://github.com/jamovi/jamovi-compiler/blob/master/schemas/optionschemas.yaml#L43